openstack/cinder/templates/backup_deployment.yaml

```diff
@@ -55,14 +55,10 @@ template: |{{`
             value: cinder-volume-backup-vmware-{{ name }}
           livenessProbe:
             exec:
-              command:
-              - bash
-              - -e
-              - -c
-              - source <(grep -Pzo '\[keystone_authtoken\][^[]*' /etc/cinder/cinder.conf | tr -d '\000' | while read var equal val; do test "$equal" = "=" && echo export OS_${var^^}=${val} ; done); export OS_IDENTITY_API_VERSION=3; test "$(openstack volume service list -f value --host "cinder-backup-{{ name }}" -c State)" = "up"
+              command: ["openstack-agent-liveness", "--component", "cinder", "--config-file", "/etc/cinder/cinder.conf", "/etc/cinder/cinder-backup.conf"]
```

Comment:
Does this work or does it need another `--config-file` in between?

Reply:
fixed